scripts/winesapos-install.sh

```diff
@@ -795,6 +809,12 @@ elif [[ "${WINESAPOS_DE}" == "plasma" ]]; then
     chroot ${WINESAPOS_INSTALL_DIR} crudini --ini-options=nospace --set /etc/xdg/konsolerc "Desktop Entry" DefaultProfile Vapor.profile
     # Image gallery and text editor.
     pacman_install_chroot gwenview kate
+elif [[ "${WINESAPOS_DE}" == "plasma-mobile" ]]; then
+     echo "Installing the KDE Plasma Mobile DE..."
+     # Use Yay since plasma mobile is on the AUR
+     yay_install_chroot plasma-mobile plasma-nano plasma-settings plasma-dialer plasma-mobile-sounds cmake
```

Comment:
I was surprised to see that was not packaged in the Chaotic AUR repository. It turns out that they are waiting for the `plasma-dialer` package to be fixed upstream.

`https://github.com/chaotic-aur/packages/issues/3052`

That's okay, we can still add this to winesapOS. The only change is that you don't need `cmake` here. We install it much earlier on in the install process since a lot of AUR packages need it.